data/json/recipes/recipe_weapon.json

```diff
@@ -6733,5 +6733,52 @@
     ["steel_chunk", 4],
     ["scrap", 12]]
   ]
+},
+{
+  "type" : "recipe",
+  "result": "chemical_thrower",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_RANGED",
+  "skill_used": "mechanics",
+  "skills_required": [[ "gun", 3 ], [ "cooking", 3]],
+  "difficulty": 7,
```

Comment:
Fair point. Never actually got to the point where I had the chain lightning.

Althought the biggest argument for making this thing harder to craft is the fact that it will at some point defungify terrain and afaik there is nothing in the game yet, that does that.